src/installer/pkg/packaging/installers.proj

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ForceRuntimeIdentifierToBeSet>true</ForceRuntimeIdentifierToBeSet>
```

Comment:
Can we rename this to `EnsureRuntimeIdentifierSet`?

Also, can we add an MSBuild comment here for why we need to ensure that the `RuntimeIdentifier` is set so that we understand this in the future?

Reply:
I'll try. The bigger question is why do I need to only conditionally set it. I wasn't able to figure that out, but if I set it unconditionally in the Directory.Build.props the installer tests failed.

Reply:
Hmmm. What failures were you seeing with it unconditionally set?